### PR TITLE
Goldie/intelligence

### PR DESCRIPTION
**Pull request by iatkins**

### CODE REVIEW

File: server/models/GameLogic.js [hunk 1]
```diff
@@ -361,9 +361,9 @@ class GameLogic {
 
     updateLabels(word) {
         this.playedCards.forEach(card => {
-            const cardScore = this.votes.filter(vote => card.cardId === vote.cardId).length
+            const cardScore = this.votes.filter(vote => card.cardId === vote.cardId && !vote.username.includes("CPU")).length;
```
Comment (tillywoodfield) [inline server/models/GameLogic.js]: Someone may have CPU in their username, is it better to find the user and use the `user.real` flag

File: server/queries/db.js [hunk 2]
```diff
@@ -53,6 +53,29 @@ module.exports.validatePassword = (username, password) => {
...
+module.exports.addLabel = (cardId, word) => {
+    return new Promise((resolve, reject) => {
+        CardLabels.findOne({
+            where: { cardId }
+        }).then(card => {
+            const labels = card.dataValues.labels;
+            if (!labels.includes(word)) {
+                labels.push(word);
+                card.update({
+                labels: labels
```
Comment (tillywoodfield) [inline server/queries/db.js]: indent, and `labels: labels` can be simplified to `labels`

File: server/services/AI/AI.js [hunk 3]
```diff
@@ -53,3 +45,11 @@ exports.autoWord = (cardId) => {
         });
     });
 }
+
+exports.addLabel = (cardId, word) => {
+    return new Promise((resolve, reject) => {
+        db.addLabel(cardId, word);
+        resolve();
+
```
Comment (tillywoodfield) [inline server/services/AI/AI.js]: extra line

File: server/services/AI/AI.js [hunk 4]
```diff
@@ -53,3 +45,11 @@ exports.autoWord = (cardId) => {
         });
     });
 }
+
+exports.addLabel = (cardId, word) => {
+    return new Promise((resolve, reject) => {
+        db.addLabel(cardId, word);
+        resolve();
```
Comment (tillywoodfield) [inline server/services/AI/AI.js]: This should be a `.then`

File: server/services/AI/AI.js [hunk 5]
```diff
@@ -53,3 +45,11 @@ exports.autoWord = (cardId) => {
...
+
+    });
+}
```
Comment (tillywoodfield) [inline server/services/AI/AI.js]: new line

**Comment by sibela46** [comment]
Do you wanna make it so that each label is stored together with its score (which is increased every time someone plays that word), so that when the AI picks a label it's not at random but based on a priority score

**Review by tillywoodfield — APPROVED** [review]

**Pull request merged**